action.yml

```
@@ -1,29 +1,31 @@
 name: 'Interface exporter action'
-description: 'An action to generate npm packages based on Solidity contract interfaces'
+description: 'An action to generate npm packages based on solidity interfaces and contracts'
 author: 'Wonderland'
+
 inputs:
-  out_dir:
-    description: 'path to the `out` folder containing the compiled contracts'
-    required: true
-  interfaces_dir:
-    description: 'path to the `interfaces` folder containing the contract interfaces'
-    default: 'src/interfaces'
-    required: false
-  typing_type:
-    description: 'interface compatibility: abi, contracts'
-    required: true
   package_name:
-    description: 'name of the package to be publish'
-    required: true
-  destination_dir:
-    description: 'path to the directory where your package will be exported'
+    description: 'name of the package to be published'
     required: true
-  contracts_dir:
-    description: 'path to the directory where the contracts are located'
+  out:
+    description: 'path to the `out` folder containing the compiled contracts'
+    required: false
+    default: 'out'
+  interfaces:
+    description: 'path to the interfaces directory'
+    required: false
+    default: 'interfaces'
+  contracts:
+    description: 'path to the contracts directory'
+    required: false
+    default: 'contracts'
```

The idea here is to have the defaults that match the structure of the [foundry boilerplate](https://github.com/defi-wonderland/solidity-foundry-boilerplate). For repositories created from the boilerplate, no extra configuration will be required.
```suggestion
  interfaces:
    description: 'path to the interfaces directory'
    required: false
    default: 'solidity/interfaces'
  contracts:
    description: 'path to the contracts directory'
    required: false
    default: 'solidity/contracts'
```